src/python/aqueduct_executor/operators/utils/enums.py

```diff
@@ -13,6 +13,19 @@ def __contains__(cls, item: Any) -> Any:
         return item in [v.value for v in cast(Iterable[Enum], cls.__members__.values())]
 
 
+class ExecutionStatus(str, Enum, metaclass=MetaEnum):
+    UNKNOWN = "unknown"
```

Comment:
When is an execution state ever unknown? Doesn't it always start at PENDING then become either SUCCEEDED or FAILED?

Reply:
It's here mainly to be consistent with go code, and go use `unknown` internally for job status handling.

Reply:
ok I see its like a default value we use when we return an error or something